<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>176</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color : rgb(255, 255, 255);
font: 16pt &quot;나눔고딕 ExtraBold&quot;;</string>
      </property>
      <property name="text">
       <string>크롬 브라우저 버전 입력
예: 115.0.5790.98</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="3" column="1">
       <widget class="QTextEdit" name="textEdit_chromeVersion">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">background-color : rgb(255, 255, 255);
font: 16pt &quot;나눔고딕 ExtraBold&quot;;</string>
        </property>
        <property name="text">
         <string>버전 입력: </string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="manual_button">
        <property name="text">
         <string>버전확인방법</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_saveChromeVersion">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : rgb(85, 85, 85);
color: rgb(255, 255, 255);
font: 9pt &quot;나눔고딕 ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>저장</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
